azext_edge/edge/providers/orchestration/clone.py

```diff
@@ -505,17 +500,17 @@ def deploy(
             # TODO: Show warnings if they exist from federation
 
             for i in range(total_pages):
-                status = f"Replicating {deployment_name} {i+1}/{total_pages}"
+                status = f"Replicating {deployment_name} {i + 1}/{total_pages}"
                 console.update(status=status)
-                page = f"_{i+1}" if total_pages > 1 else ""
+                page = f"_{i + 1}" if total_pages > 1 else ""
                 poller = self._deploy_template(
                     content=deployment_work[i],
                     parameters=parameters,
                     deployment_name=f"{deployment_name}{page}",
                 )
                 deployment_link = self._get_deployment_link(deployment_name=f"{deployment_name}{page}")
                 DEFAULT_CONSOLE.print(
-                    f"->[link={deployment_link}]Link to {self.cluster_name} deployment {i+1}/{total_pages}[/link]",
+                    f"->[link={deployment_link}]Link to {self.cluster_name} deployment {i + 1}/{total_pages}[/link]",
```

Comment:
line 1188 `symbolic_name = f"{key.value}s_{len(deployments_by_key)+1}"` is also causing lint issue